yo1k/tic_tac_toe/test/core/test_game.py

```diff
@@ -8,14 +11,11 @@
     Board,
     Cell,
     Mark)
-from collections.abc import Sequence
-from typing import Optional
-import unittest
 
 
 class ListActionQueue(ActionQueue):
-    def __init__(self, actions: Sequence[Optional[Action]]):
-        self.actions = actions
+    def __init__(self, actions: MutableSequence[Optional[Action]]):
+        self.actions: MutableSequence[Optional[Action]] = actions
         self.i = 0
```

Comment:
The attribute type annotation is missing for `self.i`.

Reply:
Done.